src/de/cuuky/varo/game/world/setup/AutoSetup.java

```diff
@@ -128,4 +130,40 @@ private void setupSpawns(Location middle) {
             new AutoStart(start);
         }
     }
-}
\ No newline at end of file
+
+    private Location getLobbyLocation(World world, long x, long z) {
```

Comment:
Why are you using `long` instead of `int`?

Reply:
I intended to use the type that is also used by the `Location` class. I mistakenly thought that it was a `long`. I looked it up and saw that a `double` is used. I will correct this.

Reply:
There should be no need for using `double` as we are only working with full blocks.